test/integration/sds_dynamic_integration_test.cc

```diff
@@ -111,6 +111,9 @@ class SdsDynamicIntegrationBaseTest : public Grpc::BaseGrpcClientIntegrationPara
 
   Network::Address::IpVersion ipVersion() const override { return GetParam().ip_version; }
   Grpc::ClientType clientType() const override { return GetParam().sds_grpc_type; }
+  virtual std::unique_ptr<FakeUpstream>& sdsUpstream() { return fake_upstreams_[0]; }
+  // Index in fake_upstreams_ vector of the data plane upstream.
+  int upstreamIndex() { return 1; }
```

Comment:
hm, arguably 0 is not an upstream and 1 is, but given they're all in fake_upstreams_ I think the naming here is a bit ambiguous.  maybe dataPlaneUpstream[Index] just for clarity?

Reply:
Good point, updated to dataPlaneUpstreamIndex.